test/test_automated_install.py

```diff
@@ -189,107 +189,6 @@ def test_installPiholeWeb_fresh_install_no_errors(Pihole):
     assert 'index.js' in web_directory
     assert 'blockingpage.css' in web_directory
 
-def test_installPiholeWeb_empty_directory_no_errors(Pihole):
```

Comment:
You should keep a modified version of this first test

Reply:
The output will be the same no matter what at this stage, I took out the checks for that directory/the files